amazon-kinesis-client/src/main/proto/messages.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto2";
+package software.amazon.kinesis.retrieval.kpl;
+
+message Tag {
```

Comment:
Is this a subset of messages kcl cares about copied from here - https://github.com/awslabs/amazon-kinesis-producer/blob/master/aws/kinesis/protobuf/messages.proto ?

Reply:
Yes.

Reply:
As I mentioned in another comment, ideally these would be in a shared repo... but this is better than the previous setup I think.

Reply:
Can we please add a comment "Copied from amazon-kinesis-producer/aws/kinesis/protobuf/messages.proto with subset of messages that KCL needs" or something along those lines

Reply:
Done